src/main/resources/com/puppycrawl/tools/checkstyle/meta/checks/coding/IllegalInstantiationCheck.xml

```diff
@@ -32,7 +32,7 @@
  There is a limitation that it is currently not possible to specify array classes.
  &lt;/p&gt;</description>
          <properties>
-            <property default-value="" name="classes" type="java.lang.String[]">
+            <property default-value="{}" name="classes" type="java.lang.String[]">
```

Comment:
https://checkstyle.sourceforge.io/checks/coding/illegalinstantiation.html#Properties

this is nuance .....
we render `{}` as in web we  can not show empty value. Users need to see something.
But in this files it, very technical files, it should stay as before  `default-value=""`